drivers/sensor/lis2dh/lis2dh.c

```diff
@@ -424,12 +430,83 @@ int lis2dh_init(const struct device *dev)
 		    cfg->bus_name, 1 << (LIS2DH_FS_IDX + 1),
 		    LIS2DH_ODR_IDX, (uint8_t)LIS2DH_LP_EN_BIT, lis2dh->scale);
 
+#ifdef CONFIG_PM_DEVICE
+	lis2dh->power_state = PM_DEVICE_STATE_ACTIVE;
+	lis2dh->reg_ctrl1_active_val = LIS2DH_ACCEL_EN_BITS | LIS2DH_LP_EN_BIT |
+				       LIS2DH_ODR_BITS;
```

Comment:
Once that you set power_state to active maybe it is unnecessary to init also reg_ctrl1_active_val. This field will be initialized the first time sensor will be suspended. I'm not blocking the PR for this anyway.

Reply:
Agreed, not necessary.